src/styles/index.css

```diff
@@ -0,0 +1,143 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+}
+.header__logo {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: #060b35;
+  width: 40px;
+  height: 40px;
+}
+.header__nav-list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  color: #060b35;
+  text-decoration: none;
+  transition: color 300ms;
+}
+.header__nav-link:hover {
+  color: #00acdc;
+}
+.header__nav .is-active {
+  border-bottom: 2px solid #00acdc;
+}
+
+.card {
+  width: 200px;
+  padding: 20px;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  font-family: Arial, sans-serif;
+  text-align: center;
+}
```

Comment:
The hover effect to increase the card size by 20% is missing. According to the task requirements, you should implement a hover effect that scales the card size by 20% when hovered over. Consider using CSS transform properties to achieve this effect.